backend/src/main/java/gov/cdc/usds/simplereport/service/idverification/LiveExperianService.java

```diff
@@ -184,18 +184,7 @@ public IdentityVerificationAnswersResponse submitAnswers(
       }
 
       boolean passed;
-      try {
-        // find overall decision ("CrossCore 2.x Technical Developer Guide.pdf" page 28-29)
-        String decision = findNodeInResponse(responseEntity, PID_OVERALL_DECISION_PATH).textValue();
-        // if experian responds with ACCEPT, we will consider the id verification successful
-        passed = SUCCESS_DECISION.equals(decision);
-      } catch (ExperianNullNodeException e) {
-        // Experian does not always return the overall decision. If this happens, check for the
-        // value of "final decision".
-        String finalDecision =
-            findNodeInResponse(responseEntity, PID_FINAL_DECISION_PATH).textValue();
-        passed = SUCCESS_DECISION_SHORT.equals(finalDecision);
-      }
+      passed = hasPassed(responseEntity);
```

Comment:
can you combine declaration and assignment into one line?
`boolean passed = hasPassed(responseEntity);`